### PR TITLE
Tpwdcee 6739 configuration

### PR DESCRIPTION
**Pull request by tatsta**

### CODE REVIEW

File: src/PhpVersion.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    def get_tested_php_versions_from_config(self) -> list:
+        """
+        Returns tested php versions from config
+        :return: list
+        """
+        return self.tested_php_versions_from_config
+
+    def set_tested_php_versions_from_changelog(self):
+        pass
```
Comment (flavius) [inline src/PhpVersion.py:52]: what's the goal with the empty methods?

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    args = parser.parse_args()
+    try:
+        extension_name = Constants.EXTENSION_NAMING_CONVENTION[args.repository]
+    except KeyError:
+        raise Exception("Unknown extension name {}".format(args.repository))
+    action = args.action
+    if args.action == "initial_changelog_and_version_update":
+        add_new_changelog_entry_and_update_internal_files(extension_name)
+
+#   if action == check_changelog_updated
```
Comment (flavius) [inline src/main.py:69]: part of a follow-up ticket or forgotten code?

File: src/PhpVersion.py (hunk 1 above)
Comment (tatsta) [reply src/PhpVersion.py:52]: They are preparation for the follow up US

File: src/main.py (hunk 2 above)
Comment (tatsta) [reply src/main.py:69]: This is preparation for the follow up US

File: src/tests/test_data/.bin/compatible-shop-releases.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+3.8.0
```
Comment (brkicadis) [inline src/tests/test_data/.bin/compatible-shop-releases.txt:1]: why is this part of the action? this was just for testing, or?

File: src/tests/test_data/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
+sudo: false
```
Comment (brkicadis) [inline src/tests/test_data/.travis.yml:1]: same here

File: src/tests/test_data/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Changelog
```
Comment (brkicadis) [inline src/tests/test_data/CHANGELOG.md:1]: same here

File: src/tests/test_data/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Wirecard WooCommerce Extension
```
Comment (brkicadis) [inline src/tests/test_data/README.md:1]: same here?

File: src/tests/test_data/wirecard-woocommerce-extension/readme.txt [hunk 7]
```diff
@@ -0,0 +1,88 @@
+=== Wirecard WooCommerce Extension ===
```
Comment (brkicadis) [inline src/tests/test_data/wirecard-woocommerce-extension/readme.txt:1]: same here?

File: src/tests/test_data/wirecard-woocommerce-extension/woocommerce-wirecard-payment-gateway.php [hunk 8]
```diff
@@ -0,0 +1,398 @@
+<?php
```
Comment (brkicadis) [inline src/tests/test_data/wirecard-woocommerce-extension/woocommerce-wirecard-payment-gateway.php:1]: same here?

File: src/tests/test_data/.bin/compatible-shop-releases.txt (hunk 3 above)
Comment (tatsta) [reply src/tests/test_data/.bin/compatible-shop-releases.txt:1]: all files from test data - are for testing purposes. They are also used in unit tests

**Pull request merged**